application/bootstrap.php

```diff
@@ -63,7 +63,8 @@
 /**
  * Set the default language
  */
-I18n::lang('en-us');
+	I18n::lang('en-en');
```

Comment:
I don't think `en-en` is a thing. This should be `en-us` or just `en` (if kohana accepts that)